### PR TITLE
[hi] Add Hindi glossary entry for "kube-controller-manager"

### PR DESCRIPTION
**Pull request by CloudShiva7cr**
Localizes kube-controller-manager.md in Hindi

### Description

#Fixes Localizes content/en/docs/reference/glossary/kube-controller-manager.md in Hindi 
content/hi/docs/reference/glossary/kube-controller-manager.md

### CODE REVIEW

File: content/hi/docs/reference/glossary/kube-controller-manager.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: क्यूब-कंट्रोलर-मैनजर (kube-controller-manager)
```
Comment (utkarsh-singh1) [inline content/hi/docs/reference/glossary/kube-controller-manager.md:2]: put kube-controller-manager in aka.

Comment (CloudShiva7cr) [reply content/hi/docs/reference/glossary/kube-controller-manager.md:2]: Hi @utkarsh-singh1 I didn't get this ? Can you elaborate more? How inside "aka" ?

**Comment by CloudShiva7cr** [comment]
@niranjandarshann @utkarsh-singh1 Can you review once, have made necessary changes as per your suggestions,  Thanks!!

File: content/hi/docs/reference/glossary/kube-controller-manager.md (hunk 1 above)
Comment (utkarsh-singh1) [reply content/hi/docs/reference/glossary/kube-controller-manager.md:2]: There is `aka:` key below, where you can put alternate translation word in the title.

**Pull request closed without merging**